.github/workflows/main.yml

```diff
@@ -20,6 +20,6 @@ jobs:
       - name: Run linter
         run: flake8 turbo
       - name: Run formatter
-        run: black --target-version=py38 --check .
+        run: black --target-version=py38 --check .  --exclude doc
```

Comment:
we can also add configuration to `pyproject.toml` rather than keep it in the CI file. Probably for another PR, though. Just a note here :)